### PR TITLE
TiFlash Late Materialization GA

### PR DESCRIPTION
**Pull request by Lloyd-Pottiger**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-late-materialization.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ summary: 介绍通过使用 TiFlash 延迟物化的方式来加速 OLAP 场景
 
 > **警告：**
```
Comment (qiancai) [inline tiflash/tiflash-late-materialization.md:8]:
```suggestion
> **注意：**
```

File: tiflash/tiflash-late-materialization.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ summary: 介绍通过使用 TiFlash 延迟物化的方式来加速 OLAP 场景
...
 >
-> 该功能目前是实验性功能，其形式和使用方法可能会在未来版本中发生变化。
+> 在 TiFlash 存算分离架构和 Fast Scan 模式下，延迟物化功能暂不可用。
```
Comment (qiancai) [inline tiflash/tiflash-late-materialization.md:10]:
```suggestion
> 在 TiFlash [存算分离架构](/tiflash/tiflash-disaggregated-and-s3.md)和 [Fast Scan 模式](/tiflash/use-fastscan.md)下，延迟物化功能暂不可用。
```

File: system-variables.md [hunk 3]
```diff
@@ -2593,7 +2593,7 @@ mysql> desc select count(distinct a) from test.t;
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
 - 类型：布尔型
-- 默认值：`OFF`
+- 默认值：`ON`
 - 这个变量用来控制是否启用 [TiFlash 延迟物化](/tiflash/tiflash-late-materialization.md)功能。
 - 默认情况下，如果 `SELECT` 语句中包含过滤条件（`WHERE` 子句），TiFlash 会先扫描查询所需列的全部数据后再进行过滤。当设置该变量为 `ON` 开启 TiFlash 延迟物化功能时，TiFlash 可以先扫描过滤条件相关的列数据，过滤得到符合条件的行后，再扫描这些行的其他列数据，继续后续计算，从而减少 IO 扫描和数据处理的计算量。
```
Comment (qiancai) [inline system-variables.md:2598]: 这里需要调整下描述，因为默认情况下，TiFlash 延迟物化功能是开启的

File: system-variables.md [hunk 4]
```diff
@@ -2593,9 +2593,9 @@ mysql> desc select count(distinct a) from test.t;
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
 - 类型：布尔型
-- 默认值：`OFF`
-- 这个变量用来控制是否启用 [TiFlash 延迟物化](/tiflash/tiflash-late-materialization.md)功能。
-- 默认情况下，如果 `SELECT` 语句中包含过滤条件（`WHERE` 子句），TiFlash 会先扫描查询所需列的全部数据后再进行过滤。当设置该变量为 `ON` 开启 TiFlash 延迟物化功能时，TiFlash 可以先扫描过滤条件相关的列数据，过滤得到符合条件的行后，再扫描这些行的其他列数据，继续后续计算，从而减少 IO 扫描和数据处理的计算量。
+- 默认值：`ON`
+- 这个变量用来控制是否启用 [TiFlash 延迟物化](/tiflash/tiflash-late-materialization.md)功能。注意在 TiFlash [存算分离架构](/tiflash/tiflash-disaggregated-and-s3.md)和 [Fast Scan 模式](/tiflash/use-fastscan.md)下，延迟物化功能暂不可用。
+- 当设置该变量为 `OFF` 关闭 TiFlash 延迟物化功能时，如果 `SELECT` 语句中包含过滤条件（`WHERE` 子句），TiFlash 会先扫描查询所需列的全部数据后再进行过滤。当设置该变量为 `ON` 开启 TiFlash 延迟物化功能时，TiFlash 可以先扫描下推到 TableScan 算子的过滤条件相关的列数据，过滤得到符合条件的行后，再扫描这些行的其他列数据，继续后续计算，从而减少 IO 扫描和数据处理的计算量。
```
Comment (qiancai) [inline system-variables.md:2598]:
```suggestion
- 当设置该变量为 `OFF` 关闭 TiFlash 延迟物化功能时，如果 `SELECT` 语句中包含过滤条件（`WHERE` 子句），TiFlash 会先扫描查询所需列的全部数据后再进行过滤。当设置该变量为 `ON` 开启 TiFlash 延迟物化功能时，TiFlash 会先扫描下推到 TableScan 算子的过滤条件相关的列数据，过滤得到符合条件的行后，再扫描这些行的其他列数据，继续后续计算，从而减少 IO 扫描和数据处理的计算量。
```

**Review by qiancai — APPROVED** [review]
other LGTM

**Review by flowbehappy — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**